mozilla/bedrock
# Initial work on translation bar. Work in progress.

## One-line summary

Show translation bar if we detect that the user viewing a page preferred a language in which the page was available, but which wasn't the current translation being viewed. 

## Issue

#6857 

## Screenshots

*WIP*

## Checklist

- [x] Remove svelte
- [x] Add CSS styles
- [x] Add localization
- [ ] Add bleach to backend code for strings

## Testing

- Tweak your browser accept language settings as needed.
- Visit a page that isn't your preferred language set above, e.g. localhost:8000/de/firefox/new/
- Check the translation bar offers the preferred language and takes you there on click.
